src/Service/Font/Service_Font.re

```diff
@@ -35,15 +35,15 @@ type msg =
   | FontLoaded(t)
   | FontLoadError(string);
 
-let requestId = ref(0);
-
-let setFont = (dispatch1, fontFamily, fontSize, smoothing) => {
+let setFont = (requestId, dispatch1, fontFamily, fontSize, smoothing) => {
```

Comment:
`dispatch1`? What does that mean?

Also seems like some labelled arguments might be good here. There's quite a few arguments now, and they're not very specifically typed.

Reply:
Ah, I think `dispatch1` was left over from porting from the previous font strategy over to `Service_Font`. Not sure why it has the `1` - I renamed it and change all the arguments to labelled.